pyHalo/preset_models.py

```diff
@@ -259,14 +261,18 @@ def SIDM(z_lens, z_source, cross_section_name, cross_section_class, kwargs_cross
 def ULDM(z_lens, z_source, log_mlow=6., log_mhigh=10., b_uldm=1.1, c_uldm=-2.2,
                   c_scale=15., c_power=-0.3, cone_opening_angle_arcsec=6.,
                   sigma_sub=0.025, LOS_normalization=1., log_m_host= 13.3, power_law_index=-1.9, r_tidal='0.25Rs',
-                  mass_definition='ULDM', log10_m_uldm=-22, uldm_plaw=1/3, scale_nfw=False,**kwargs_other):
+                  mass_definition='ULDM', log10_m_uldm=-22, uldm_plaw=1/3, scale_nfw=False, flucs=False, 
```

Comment:
For these functions, I like having the essential parameters relevant to the model be required arguments (for example, log_mc, or the half-mode mass, is the (only) required argument for the warm dark matter (WDM) preset model. Could you make log10_m_uldm, a required argument, and also set the default value flucs=True?

Reply:
In addition, the compute the de Broglie wavelength we need a velocity scale, which we will want to sample in our analysis. could you add an argument (immediately following the particle mass) with a default value of 200 km/sec? so the function will look like ULDM(z_lens, z_source, log10_m_uldm, velocity_scale=200, ...)